examples/liquid.ini

```diff
@@ -251,6 +251,21 @@ wait_green_count = 6
 # - `image_classification_classify_images_model`   - Set the model used for image classification. Default is "mobilenet".
 #                                                    Options are "mobilenet", "resnet", "inception" and "densenet".
 # for documentation of the image classification settings see: https://github.com/liquidinvestigations/image-classification-service
+# NLP service for named entity recocnition
+# - `nlp_entity_extraction_enabled` - Setting to enable entity extraction. Default is "false".
+# - `nlp_language_detection_enabled` - Setting to enable language detection. Default is "false".
+# - `nlp_count` - Set the number of containers to run. Default is 2
+# - `nlp_fallback_language` - language that language detection returns if no
+#                 language can be determined by service
```

Comment:
The style of the lines above this is:
- sentence starts with uppercase, ends with .
- overflow lines align with previous `-`

Let's try to keep that style here too